app/views/configuration/timeprofile_edit.html.haml

```diff
@@ -2,4 +2,4 @@
   = react('SettingsTimeProfileForm', :timeProfileId => "", :timezones => @all_timezones, :action => @timeprofile_action, :userid => @userid)
 - else
   = react('SettingsTimeProfileForm', :timeProfileId => @timeprofile.id.to_s, :timezones => @all_timezones, :action => @timeprofile_action, :userid => @userid)
-= render :partial => 'timeprofile_form'
+  = render :partial => 'timeprofile_form'
```

Comment:
we have react file, right? what is this line do?

Reply:
It calls this haml file: /app/views/configuration/_timeprofile_form.html.haml which loads the report table at the bottom of the page. If you want we can get rid of this separate haml file and add the react render function here.

Reply:
it is fine, i think it is better to have separate file for table.